dotcom-rendering/src/components/EnhanceAffiliateLinks.importable.tsx

```diff
@@ -78,24 +78,24 @@ export const EnhanceAffiliateLinks = () => {
 				? utmParamsFromReferrer
 				: '';
 
-		for (const link of allLinksOnPage) {
```

Comment:
Are the changes here related to adding the new component event? If not it might be preferable to separate them out so they're not tied to this version bump!

Reply:
Yeah I wondered about reverting this change entirely, it's just code style change now - initially made when I was planning on tracking Affiliate Links and not the Disclaimer, I'll revert as it's not relevant to the objective at hand - and adds little in terms of code improvements!